src/searchReplace.tsx

```diff
@@ -294,6 +312,21 @@ const SearchReplaceElement = (props: IProps) => {
         >
           <refreshIcon.react></refreshIcon.react>
         </Button>
+        <Button
```

Comment:
You should `disabled` the button if there is no results.

To disable, add the property `disabled={condition}` to the button.